src/library/FOSSBilling/Mail.php

```diff
@@ -16,24 +17,8 @@
 use Symfony\Component\Mailer\Transport;
 use Symfony\Component\Mime\Address;
 
-class Mail implements InjectionAwareInterface
+class Mail
 {
-    protected ?\Pimple\Container $di = null;
-
-    private Email $email;
-    private ?string $transport = null;
-    private ?string $dsn = null;
```

Comment:
We kinda need these :P

Reply:
They happened to be between the `$di` statements 👀 - fixed.